### PR TITLE
remove more unused TOKs

### PR DESCRIPTION
**Pull request by WalterBright**

### CODE REVIEW

File: test/fail_compilation/ice17074.d [hunk 1]
```diff
@@ -1,19 +1,13 @@
 /*
-TEST_OUTPUT:
```
Comment (dkorpel) [inline test/fail_compilation/ice17074.d:2]: Removing an expected error from the test suite doesn't look like refactoring

Comment (WalterBright) [reply test/fail_compilation/ice17074.d:2]: It's not an expected error. __overloadSet is never used.

Comment (ibuclaw) [reply test/fail_compilation/ice17074.d:2]: To put it another way, `__overloadset` is not a D keyword (though the double-underscore means that it is reserved for the compiler to claim or not whenever it wishes).

Comment (RazvanN7) [reply test/fail_compilation/ice17074.d:2]: Essentially, this PR frees __overloadset, so technically, I guess it's ok to modify this test.

**Review by RazvanN7 — APPROVED** [review]

**Pull request merged**